SAP2000_Adapter/CRUD/Create/Loads.cs

```diff
@@ -425,6 +425,29 @@ private bool CreateLoad(GravityLoad gravityLoad)
         }
 
         /***************************************************/
-        
+
+        private bool CreateLoad(BarTemperatureLoad bhLoad)
+        {
+            List<Bar> bars = bhLoad.Objects.Elements.ToList();
+            string loadPat = bhLoad.Loadcase.CustomData[AdapterIdName].ToString();
+            double tempChange = bhLoad.TemperatureChange;
+            int loadType = 1; // BHoM currently only supports uniform temperature change (=1); SAP supports gradients in local 2 and 3 axes (=2 and =3).
+            bool replace = true;
+            
+            foreach (Bar bar in bars)
+            {
+                string name = bar.CustomData[AdapterIdName].ToString();
+                bool replaceNow = replace;
```

Comment:
should be able to leave off replaceNow - only necessary for looping over directions when you've got a 3d load.